contribution-guide.md

```diff
@@ -0,0 +1,154 @@
+   - language specification: <a href="https://github.com/ballerina-platform/ballerina-spec/issues">ballerina-spec</a> repo.
+   - website: <a href="https://github.com/ballerina-platform/ballerina-dev-website/issues">ballerina-dev-website</a> repo. 
+
+-  Start with GitHub issues that can be fixed easily:
+    - Browse issues labeled "good first issue" in the <a href="https://github.com/ballerina-platform/ballerina-lang/issues">ballerina-lang</a> repo.
+    - Use comments on the issue itself to indicate that you will be working on it and get guidance and help.
+
+## Set up the prerequisites
+1. Download [Ballerina](https://ballerina.io) and go through the [learning resources](https://ballerina.io/learn).
+2. Read the [Ballerina Code of Conduct](https://ballerina.io/code-of-conduct).
+3. Download the Java SE Development Kit (JDK) version 8 from one of the following locations:
+    - [Oracle](http://www.oracle.com/technetwork/java/javase/downloads/index.html)
+    - [OpenJDK](http://openjdk.java.net/install/index.html) and install it.
+>**Note:** Set the JAVA_HOME environment variable to the path name of the directory into which you installed JDK.
+4. Download [Node.js](https://nodejs.org/en/download/) (version 8.9.x or the latest LTS release) and install it.
+5. Download [npm](https://www.npmjs.com/get-npm) (version 5.6.0 or later) and install it.
+
+## Set up the developer environment
+
+See the below sections to set up IntelliJ IDEA or Eclipse as your development environment.
+
+### Set up IntelliJ IDEA
+
+Follow the steps below to import the Ballerina project to IntelliJ IDEA similar to any other Gradle project.
+
+1. In IntelliJ, click **File** and then click **Open**.
+2. Browse the file system and select the <BALLERINA-LANG_REPO> root directory.
+>**Tip:** If you are opening the directory for the first time in IntelliJ and if prompted to import it as a Gradle project, select the **Auto import** option.
+3. Set Project SDK as Java 1.8.
+4. Install the following plugins:
+    - [Ballerina plugin for IntelliJ IDEA](https://plugins.jetbrains.com/plugin/9520-ballerina)
+    
+        For installation instructions, see [Ballerina IntelliJ Plugin Documentation](https://ballerina.io/learn/tools-ides/intellij-plugin).
+    - [ANTLR plugin for IntelliJ IDEA](https://plugins.jetbrains.com/plugin/7358-antlr-v4-grammar-plugin)
+        
+        This will be useful to check and validate a [grammar rule](#contributing-to-ballerina-grammar) you wrote.
+
+### Set up Eclipse
+
+As all Ballerina repositories are developed as Maven or Gradle projects, follow the steps below to import any Ballerina project to Eclipse similar to any Maven or Gradle project.
+
+1. In Eclipse, click **File** and then click **Import..**.
+2. Under the **Maven** option, select **Existing Maven Projects**. 
+3. Browse the file system and select the Maven module to open it.
```

Comment:
What is teh reason for skipping the section [1] for ANTLR plugin installation? Since it is explained for IDEA plugin, either we should do the same for eclipse as well or move it to a common section 

[1] https://github.com/ballerina-platform/ballerina-lang/blame/master/CONTRIBUTING.md#L130:L143

Reply:
This  is not fixed?

Reply:
Discussed with @nadeeshaan and changed the sections as follows:

- [Set up the developer environment](#set-up-the-developer-environment)
    - [Set up the Ballerina plugin](#set-up-the-ballerina-plugin)
    - [Set up the ANTLR4 plugin](#set-up-the-antlr4-plugin)